src/Controller/SecurityController.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+final class SecurityController extends AbstractController
+{
+    /**
+     * @Route("/login", name="app_login")
+     */
+    public function login(AuthenticationUtils $authenticationUtils): Response
```

Comment:
We tends to use ADR, so one function per controller, that we name handler usually. The AuthenticationUtils would be injected in the constructor then, rather than in the method (during object creation rather than depending on the dispatcher magic).